docs/sources/clients/promtail/installation.md

```diff
@@ -37,7 +37,7 @@ helm repo update
 Finally, Promtail can be deployed with:
 
 ```bash
-$ helm upgrade --install promtail grafana/promtail --set "loki.serviceName=loki"
+helm upgrade --install promtail grafana/promtail --set 'config.clients[0].url=http://loki:3100/loki/api/v1/push'
```

Comment:
isn't better to have this by default?

Reply:
Indeed. I've removed the `--set` argument to keep it clean as you suggested, it will default to `http://loki-gateway/loki/api/v1/push`. Ref: https://github.com/grafana/helm-charts/blob/main/charts/promtail/values.yaml#L330-L331